### PR TITLE
Feedback area: adding a11y assessment performed by CRA

### PR DESCRIPTION
**Pull request by Garneauma**
related to : WET-456

### CODE REVIEW

**Review by duboisp — CHANGES_REQUESTED** [review]
See inline comments

File: sites/feedback/reports/a11y-3.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    },
+    {
+      "earl:test": "WCAG21:perceivable",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [
+        {
+          "@type": "acr:AttachmentImage",
+          "acr:content": {
+            "@type": "xsd:anyURI",
```
[the remark this answers is not in the document]
Comment (Garneauma) [reply sites/feedback/reports/a11y-3.json:83]: The document from CRA refers to this image as "OS Magnification".

File: sites/feedback/reports/a11y-3.json [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
...7KCi/2LsWxIZ1Ze9OLep3cEPb3GprWnmLnEk+seH0WBR3l+rCmTl3Sm5tYbzH0Mnz6+4ovnYnaMLEgBhzgkyFWZxosvcERuvE337qGuuG3oRL3SxlrPUbLlqoCacoh+Hqawp2weeB4ZT2OXjHL890Hojy/Pca21C3PTZWFwv8Usj775/niX12fGMKxVHuIYH5qK0fbkeUrReNFJdj6LCmxylyDXSbSvVRhru5ykrNCbjgtPsO3WS57KTb4pGPIxkg+R74l49xH5j09G9Of9bVHxdEY2HUh6h5W0GvbAHPSQLdJjO5d7FaBXGqkvuu64CSnQfiyTLr7FUNlPBaEtbVAJ7ElcsB5Vh61jzlEf7gW/NwefPhuz+iN4FxlJxIfPfJCdxvInX7yHcWKmPRUz/xiEDuTZ8rGsTZOMSqbyjRmJJUmfey4Tt/niLSm+8J6Q0yM5Od6ZadO7Me3CK66YIo1GIHMjG8bXJqNDgij5jPzVSZN+M1PWB0ctTfGTQei8LlhMP8agR99mQj88OIH6FXLGSOeIpXU3Gv0/mgltFS8hhcU+Uy+94ZEpvdk+q0dUIH1q2+/S2qRKR9nBS1ngDgEmY4j27q9hbKt+pHMcQ92Bp/luJvSCwgwpGzMInZyGI+b/fiZ0Q0lj9NV0jUg25AQzb8WYCb19InPzZKCU5P+HayHnqInziKPMcfvE+89i/dAjnNo7F5cRG6m+xItTT97w2e8g/gdGcfzETo49fyPeEYTfwR2sqTiFqXU2cfBKsMgmfvXqLFfGN6aHnXkpokRZi1Lr11hmQtdKLXUbD5RaUrDcSTZeiTETelIvFiSIR7GYMqaKMRPa5ARveP9wB/tbVaKFHDcvtZFJGEXdDrc5E2Qe7T4FnuZ2h7qMlOPfrYcdAwubpMQvM5jyCx5wwN8cYjSCwEFX9nFs1njmDh3I8OFDGDh0KJPnr2bjpQCTDCZ531wVPfRlYEV7EsW1x6LYYIqteswx03o3Utab81wa0Zp+ca3/fCZ2jsakHH6MWbeMT0kam2HDSWJDZ9NDoG82TCS0JGuBQyw/Gv12y3OebxzLhgLJcJE0xuxec1pDxz0oWfUsu+9GmseBL1482dCMHf0yMnr2NObcFo0bneXTI95dnsrkLoVJIn3FnN/ou43IWXgHy09FjwvhvLkwjP19ElI3R1Ic87Wn1vDtrLn0nMdyD
```
Comment (duboisp) [inline sites/feedback/reports/a11y-3.json:130]: That property `selected` is noise added by my prototype tool, please remove.

**Review by duboisp — CHANGES_REQUESTED** [review]
See inline comments.

I suggest in parenthesis to indicate the slide number in the presentation that are matching the item.

**Review by duboisp — APPROVED** [review]
Looks good, but there is some additional comments at the end, should we include those?

**Comment by duboisp** [comment]
Note: Ceci est seulement du contenu.

**Pull request merged**